src/cursor.c

```diff
@@ -168,7 +177,12 @@ static void CursorList_IncrCounter(CursorList *cl) {
  * a stuck client and a crashed server
  */
 static uint64_t CursorList_GenerateId(CursorList *curlist) {
-  uint64_t id = lrand48() + 1;  // 0 should never be returned as cursor id
+  uint64_t id = (curlist->is_coord ? rand_even48() : rand_odd48()) + 1;  // 0 should never be returned as cursor id
```

Comment:
We can do a `-2` in the case `rand_odd48` returns 2^31-1

Reply:
No need for -2 see explanation above.

Reply:
OK